docs/cli.md

```
@@ -0,0 +1,862 @@
+
+Usage: `hack global <subcommand>`
+
+Subcommands:
+
+| Subcommand | Summary |
+| --- | --- |
+| `install` | Bootstrap `~/.hack` and start Caddy + Grafana/Loki/Alloy |
+| `up` | Start global infra containers |
+| `down` | Stop global infra containers |
+| `status` | Show status for global infra (containers + networks) |
+| `logs` | Tail global infra logs (caddy|grafana|loki|alloy) |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 2; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)